mw_url_rewrite_test.go

```diff
@@ -510,3 +510,68 @@ func TestRewriterTriggers(t *testing.T) {
 		})
 	}
 }
+
+func TestInitTriggerRx(t *testing.T) {
+	testRewriteMW := &URLRewriteMiddleware{
+		BaseMiddleware: BaseMiddleware{
+			Spec: &APISpec{
+				APIDefinition: &apidef.APIDefinition{
+					VersionData: struct {
+						NotVersioned   bool                          `bson:"not_versioned" json:"not_versioned"`
+						DefaultVersion string                        `bson:"default_version" json:"default_version"`
+						Versions       map[string]apidef.VersionInfo `bson:"versions" json:"versions"`
+					}{
+						NotVersioned:   true,
+						DefaultVersion: "Default",
+						Versions: map[string]apidef.VersionInfo{
+							"Default": {
+								ExtendedPaths: apidef.ExtendedPathsSet{
+									URLRewrite: []apidef.URLRewriteMeta{
+										{
+											Triggers: []apidef.RoutingTrigger{
+												{
+													Options: apidef.RoutingTriggerOptions{
+														HeaderMatches: map[string]apidef.StringRegexMap{
```

Comment:
I like indentation level

Reply:
anything we can do about this??

Reply:
I got too excited with my 4K monitor. I will try to reformat it a little bit